streaming-bytestring.cabal

```diff
@@ -235,6 +235,7 @@ test-suite test
   build-depends:
       base                  >=4        && <5
     , bytestring
+    , resourcet             >=1.2
```

Comment:
You probably noticed that the `resourcet` version bound looks too high for some of the older LTS snapshots in the test matrix...

Reply:
Yes, I was waiting for CI to fail and tell me :)